springfield/base/templates/includes/navigation/menus/resources.html

```diff
@@ -7,7 +7,7 @@
  {% set utm_params = 'utm_source=www.firefox.com&utm_medium=referral&utm_campaign=nav&utm_content=resources' %}
 
  <li class="m24-c-menu-category mzp-has-drop-down mzp-js-expandable">
-   <a class="m24-c-menu-title" href="#TODO" aria-haspopup="true" aria-controls="m24-c-menu-panel-resources" data-testid="m24-navigation-link-resources">
+   <a class="m24-c-menu-title" href="{{ url('firefox.more.index') }}" aria-haspopup="true" aria-controls="m24-c-menu-panel-resources" data-testid="m24-navigation-link-resources">
```

Comment:
Hmh I actually quite like /releases index there instead:

```suggestion
   <a class="m24-c-menu-title" href="{{ url('firefox.releases.index') }}" aria-haspopup="true" aria-controls="m24-c-menu-panel-resources" data-testid="m24-navigation-link-resources">
```